js/src/contracts/registry.js

```diff
@@ -89,7 +97,7 @@ export default class Registry {
       return instance.getAddress.call({}, [sha3, 'A']);
     })
     .then((address) => {
-      console.log('lookupAddress', name, sha3, address);
+      console.log('[lookupAddress]', `(${sha3.slice(0, 5)}...${sha3.slice(-3)}) ${name}: ${address}`);
```

Comment:
Actually revert this (at least the slicing), this does get used in management via contract.